content/1.7/back-office/dashboard/dashboard-configure.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Dashboard Configure
+weight: 1
+---
+
+# Dashboard configure
+
+3 blocks are displayed on this page:
+
+ - **Average bank fees per payment method**
+ - **Average shipping fees per shipping method**
+ - **Other settings**
+
+## Demo mode
+
+ - **Demo mode**: This toggle allows to enable or disable fake data. By default, it is disabled. When enabled, fake data are used. When disabled, the real store's data are used. The store's data are gathered by PrestaShop's statistics system.
+The following tooltip is displayed: "This mode displays sample data so you can try your dashboard without real numbers."
+
+## Average bank fees per payment method
+
+4 default fields are displayed in this block, if more payment methods are added to the shop, they will be displayed additionally here:
+
+ - **Bank transfer** - An help text is displayed under the field: "Choose a fixed fee for each order placed in USD% with Bank transfer." Only numbers are allowed in this field. On the right of this field, the default currency (defined in International > Localization) is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Bank transfer field is invalid."
+ - **Bank transfer** - An help text is displayed under the field: "Choose a variable fee for each order placed in USD% with Bank transfer. It will be applied on the total paid with taxes." Only numbers are allowed in this field. On the right of this field, a percentage sign is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Bank transfer field is invalid.".
+ - **Payments by check** - An help text is displayed under the field: "Choose a fixed fee for each order placed in USD% with Payments by check.". Only numbers are allowed in this field. On the right of this field, the default currency (defined in International > Localization) is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Payments by check field is invalid."
+ - **Payments by check** - An help text is displayed under the field: "Choose a variable fee for each order placed in USD% with Payments by check. It will be applied on the total paid with taxes.". Only numbers are allowed in this field. On the right of the field, a percentage sign is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Payments by check field is invalid."
+
```

Comment:
Hello,

Could you please check the case when adding more then currency. 
@matks What do you think? 
https://watch.screencastify.com/v/iBQZ57yphNag4LjeQ3Tt

Thanks,

Reply:
@marwachelly do you need me to check or Mathieu?

Reply:
@Saimis777 Could you please check with @MatShir to identify the exact expected behavior?

Reply:
I believe this is correct, because it shows the currency which is selected in the shop, if you would change the default currency, and the currency in the dashboard would not change, then I believe it would be a problem no? @marwachelly

Reply:
@Saimis777 
Could you please check the attached  screen recording
https://watch.screencastify.com/v/ARqa5UkfTODGzueTXrgA

I think the help message doesn't match with the currency displayed also when we delete the currency added the block added is not deleted.
Could you please check with @MatShir to identify the exact expected behavior?

Thank you,

Reply:
@marwachelly @MatShir 

Ok now I see what you mean, so should this be raised as a bug? How should it be written out in the specs tho?